arxiv_test.go

```diff
@@ -109,39 +106,30 @@ func TestArxivSpider_Import(t *testing.T) {
 				"sortBy":    []string{"submittedDate"},
 				"sortOrder": []string{"descending"},
 			},
-			Error: false,
 		},
-		"failing because not arxiv": {
-			URL:            "http://medium.org/me/bookmarks",
-			ExpectedParams: map[string][]string{},
-			Error:          true,
+		"physics...": {
+			URL: "http://arxiv.org/abs/quant-ph/1234.5678v5",
+			ExpectedParams: map[string][]string{
+				"id_list":   []string{"1234.5678"},
+				"sortBy":    []string{"submittedDate"},
+				"sortOrder": []string{"descending"},
+			},
 		},
-		"failing because could not extract id": {
+		"still extracts something even if invalid id": {
 			URL:            "http://arxiv.org/abs/not-an-id",
-			ExpectedParams: map[string][]string{},
-			Error:          true,
+			ExpectedParams: map[string][]string{
```

Comment:
this one does not seem to be used in the tests anymore, though it would be better to actually test the output 😉 can you restore that?